src/transforms/push.ts

```diff
@@ -91,7 +93,7 @@ export const processPush = async (github: GitHubInstallationClient, payload: Pus
 	const log = rootLogger.child({
 		webhookId: webhookId,
 		repoName: repo,
-		orgName: owner.name,
+		orgName: owner["name"], //Less bold, still maintain it, but guess it would be empty
```

Comment:
What's the reason for the change here? Dot notation is faster to write and clearer to read, and is what we use throughout this codebase (unless we're passing a variable that changes or the property name isn't valid for dot notation).

Reply:
After adding some typings, ts complain `name` is not in the `owner` type. I check the file `src/interfaces/github.ts`, it does not have `name` under `owner`, and I check [github doc](https://docs.github.com/en/rest/repos/repos#get-a-repository), the `name` is not in the `owner` (see screenshot). So I think we have a mistake, and if there's no `name`, typescript did complain here, say you can't do `owner.name`. 

So in order to fix it at compile time, but not risk lose any information at runtime, I change it to `owner["name"]`, instead of simply remove it. 
 
<img width="351" alt="image" src="https://user-images.githubusercontent.com/105693507/182832827-fc715ca1-af9e-4143-97dd-d6e62a6f2d8f.png">

------------

Turns out the github doc is wrong, from the "schema" tab, it actually shows that `name` is there under `owner.
```json
"owner": {
      "title": "Simple User",
      "description": "Simple User",
      "type": "object",
      "properties": {
        "name": {
          "type": [
            "string",
            "null"
          ]
        },
```

So 🤷 , I revert things back. Except type it as `string | null` instead of origin in the code `string`

Tbh, I am not 100% sure which part of the github doc is wrong, although believe the "schema" is more correct. At least our code won't break with the typing `string | null`

------------

All reverted now, should be all good.